grammar.ebnf

```diff
@@ -0,0 +1,28 @@
+Module = { Declaration }, ? EOI ?;
+
+
+Declaration = StructDecl | FunctionDef;
+
+
+FunctionDef = "func", Identifier, "(",  FuncDefParamList, ")", "{",  { Statement }, "}";
+
+
+FuncDefParamList = [ParamIdentifierList, ":", Type, { ",", ParamIdentifierList, ":", Type }];
```

Comment:
You may want to have a rule for `ParamIndentifierList, ":", Type`, like `ParamSet` or smth. It's good either way, choose as you like.

Reply:
Yeah, it is much more readable your way, to be honest, I'll change it promptly.